addons/web/static/src/views/kanban/kanban_arch_parser.js

```diff
@@ -56,12 +57,24 @@ export class KanbanArchParser extends XMLParser {
         const openAction = action && type ? { action, type } : null;
         const templateDocs = {};
         const activeFields = {};
+        let headerButtons = [];
+        let buttonId = 0;
         // Root level of the template
         this.visitXML(xmlDoc, (node) => {
             if (node.hasAttribute("t-name")) {
                 templateDocs[node.getAttribute("t-name")] = node;
                 return;
             }
+            if (node.tagName === "header") {
+                headerButtons = [...node.children]
```

Comment:
list views are rng validated and the header can only contain `<button>` nodes. Kanban aren't, so even if it's not supported, people could add anything inside the header (I think, didn't check). Could you try to confirm? And probably filter out non button tags here?